src/topButton.js

```diff
@@ -0,0 +1,19 @@
+let mybutton = document.getElementById("top-btn");
+
+window.onscroll = function() {
+  scrollFunction();
+};
+function scrollFunction() {
+  if (
+    document.body.scrollTop > 385 ||
```

Comment:
How did you arrive at 385?

Reply:
I increased the length of the scroll from the top of the page until the icon appeared in the section that I wanted it to. Is there a better way to do this?

Reply:
I would probably do the calculation in JS just because you're not really guaranteed the section will be 385px down on the page, but let's call that a stretch goal

Reply:
Ok, got it.